### PR TITLE
Issue 59 - Update build instructions and update debian ssl version

### PR DESCRIPTION
**Pull request by JustinKyleJames**

### CODE REVIEW

**Comment by korydraughn** [comment]
Is this ready for review?

If not, consider converting this to draft so no one attempts to review it before it's ready.

**Comment by JustinKyleJames** [comment]
> Is this ready for review?
> 
> If not, consider converting this to draft so no one attempts to review it before it's ready.

I converted it to draft.  Terrell was going to use it to build the packages so I'll wait to see if it works for him.

These steps did work for me but in a Dockerfile.

**Comment by JustinKyleJames** [comment]
Since this hasn't been reviewed I squashed the three commits into one and forced push.

File: README.md [hunk 1]
```diff
@@ -89,12 +89,10 @@ Prerequisites
...
-	sudo yum-config-manager --enable Globus-Connect-Server-5-Stable
-	sudo yum-config-manager --enable Globus-Toolkit-6-Stable
+	wget -q https://downloads.globus.org/globus-connect-server/stable/installers/repo/rpm/globus-repo-latest.noarch.rpm
+	sudo rpm --force -i globus-repo-latest.noarch.rpm
 	sudo yum install -y globus-gridftp-server-progs globus-gass-copy-progs
-	sudo yum install -y globus-common-devel globus-gridftp-server-devel globus-gridmap-callout-error-devel
+	sudo yum --disablerepo epel install -y globus-common-devel globus-gridftp-server-devel globus-gridmap-callout-error-devel
 	sudo yum install -y libcurl-devel
 	sudo yum install -y git
 	sudo yum install -y gcc-c++
```
Comment (trel) [inline README.md:98]: could combine these...

`sudo yum install -y libcurl-devel git gcc-c++`

and could add `make` to the end - it wasn't on the vanilla container...

File: README.md [hunk 2]
```diff
@@ -89,12 +89,10 @@ Prerequisites
 
 	CentOS:
 	```
-	sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-	sudo yum install -y http://downloads.globus.org/toolkit/gt6/stable/installers/repo/rpm/globus-toolkit-repo-latest.noarch.rpm
-	sudo yum-config-manager --enable Globus-Connect-Server-5-Stable
-	sudo yum-config-manager --enable Globus-Toolkit-6-Stable
+	wget -q https://downloads.globus.org/globus-connect-server/stable/installers/repo/rpm/globus-repo-latest.noarch.rpm
+	sudo rpm --force -i globus-repo-latest.noarch.rpm
```
Comment (trel) [inline README.md:93]:
```
sudo yum install -y epel-release
sudo yum install -y https://downloads.globus.org/globus-connect-server/stable/installers/repo/rpm/globus-repo-latest.noarch.rpm
```

**Comment by korydraughn** [comment]
Looks good to me. Anything else left for this PR?

**Comment by trel** [comment]
the readme commit message mentions wget - but there is no wget in the diff.     otherwise, i think this is good/ready.

**Comment by JustinKyleJames** [comment]
> the readme commit message mentions wget - but there is no wget in the diff. otherwise, i think this is good/ready.

There was a wget added and then updated which was lost on the squash.  I will remove that note.

**Comment by trel** [comment]
i think we're good to go.   please # them both.

**Comment by JustinKyleJames** [comment]
> i think we're good to go. please # them both.

Done

**Pull request merged**